Update behavior for the ssh config/ca endpoint when a CA exists (#684)
hashicorp/web-unified-docs: content/vault/v1.17.x/content/api-docs/secret/ssh.mdx

## Submit CA information

This endpoint allows submitting the CA information for the secrets engine via an SSH

> key pair. 
> 
> <Note>
> If a certificate or key are present, an error will be returned. To update the existing
> values, the Delete CA information API will need to be called first.
> </Note>

Reviewer: ```suggestion
key pair. If you call submit CA with an existing certificate or key, the
endpoint returns an error. To update existing values, call the delete CA
information endpoint before calling the submit endpoint.
```
Edited to match other entries
